contribs/gmf/src/services/permalink.js

```diff
@@ -633,7 +645,13 @@ gmf.Permalink.prototype.handleLayersAdd_ = function(evt) {
 gmf.Permalink.prototype.handleLayersRemove_ = function(evt) {
   var layer = evt.element;
   goog.asserts.assertInstanceof(layer, ol.layer.Base);
-  this.unregisterLayer_(layer);
+  if (layer instanceof ol.layer.Group) {
+    layer.getLayers().forEach(function(l) {
+      this.unregisterLayer_(l);
+    }, this);
+  } else {
+    this.unregisterLayer_(layer);
+  }
```

Comment:
I'm still not sure if this could work for the same reasons explained in my previous comment about this.  Thoughts ?